README.md

```diff
@@ -13,6 +13,33 @@ We have a [gitter channel](https://gitter.im/exercism/xfsharp) where you can get
 
 Please see the [contributing guide](https://github.com/exercism/docs/tree/master/contributing-to-language-tracks)
 
+## Local Tools
+
+[PowerShell](https://github.com/PowerShell/PowerShell), [Fantomas](https://github.com/fsprojects/fantomas), and [FSharpLint](https://github.com/fsprojects/FSharpLint) are are available in this repo as [local tools](https://docs.microsoft.com/en-us/dotnet/core/whats-new/dotnet-core-3-0#local-tools). Example usage:
```

Comment:
Maybe be explicit about the local tools requiring .NET Core 3.0?

Reply:
Updated.